multigeiger/userdefines-example.h

```diff
@@ -1,67 +1,76 @@
-//====================================================================================================================================
 // User / hardware specific definitions - do NOT edit THIS (example) file.
 //
 // To use them please copy this file to userdefines.h (do not add that file to version control)
 // and then edit userdefines.h as appropriate for your use case / hardware.
-//====================================================================================================================================
 
-// If you use platformio, CPU is defined in platformio.ini and
-// you don't need to change the definition of CPU.
-// Arduino IDE users *must* define CPU here and uncomment one and only one of following CPU-Defines.
+
+// CPU values (board types, DO NOT CHANGE):
+#define WIFI 0  // Heltec Wifi Kit 32
+#define LORA 1  // Heltec Wifi Lora 32 (V2)
```

Comment:
This must be deleted - this board is no longer supported.
We only have WifiKit32 and Wireless Stick

Reply:
should be done after this PR (not in scope).

Reply:
#211